.github/workflows/check-go-versions.yml

```diff
@@ -25,8 +25,8 @@ jobs:
       - name: Parse officially supported Go versions
         id: parse
         run: |
-          echo "latest=${{ fromJSON(env.fetch-api-data)[0].latest }}" >> $GITHUB_OUTPUT
-          echo "penultimate=${{ fromJSON(env.fetch-api-data)[1].latest }}" >> $GITHUB_OUTPUT
```

Comment:
Instead of using the full Go versions (like 1.22.1), just grab only the "cycle" (1.22). This is what we want in the docs and in the go.mod.